lib/manageiq_foreman/lib/manageiq_foreman/paged_response.rb

```diff
@@ -2,28 +2,26 @@ module ManageiqForeman
   # like the WillPaginate collection
   class PagedResponse
     include Enumerable
+    extend Forwardable
```

Comment:
I'm ok with this, but isn't active_support avail here?  If so, the delegate method I think is already available to you.